### ISSUE TITLE
Implement Trainings opportunity landing page

### ISSUE THREAD

**Issue by hannahkane** [mozilla/network#110]
Content is [here](https://github.com/mozilla/network/issues/44#issuecomment-283819302). Layout/design should be based on the comparable Internet Health Report landing page.

 - [x] Mockup
 - [ ] Build

**Comment by hannahkane** [comment]
@gvn @alanmoo @taisdesouzalessa @sabrinang — do you think we need a mockup of this page before we can build it? It should be similar to the Internet Health Report page (#109), but with different copy.

If so, who should I assign this ticket to— @sabrinang or @taisdesouzalessa?

**Comment by taisdesouzalessa** [comment]
I'll do the mockup @gvn

**Comment by taisdesouzalessa** [comment]
@hannahkane here is the mockup:

https://invis.io/ZJ9YTAGEA#/222644515_Landing-Page-Leadership-Training

2 comments:
- It would be nice to have an image of a real training session. Who should I connect with to try to get one? If not possible, I can use a generic picture.
- where "start your training" links to?

cc: @xmatthewx @sabrinang

**Comment by hannahkane** [comment]
Thanks, @taisdesouzalessa. This is looking good!

@acabunoc or @zee-moz - do you have any photos from trainings we could look at?

I believe @acabunoc will provide the link when it's ready. It's likely to be hosted on Teachable.

**Comment by acabunoc** [comment]
Here's an album of photos:
https://www.flickr.com/photos/139954518@N06/sets/72157662003654193/

Otherwise, MozFest photos might work?

And yes, Teachable. Will provide link.

**Comment by sabrinang** [comment]
I chose two photo options from the WoW Berlin album and here it is in context:
<img width="1081" alt="screen shot 2017-03-09 at 4 17 07 pm" src="https://cloud.githubusercontent.com/assets/2492510/23776578/f7304bb6-04e3-11e7-8466-0b288363c1c5.png">

I like the left option - photos exported here: https://drive.google.com/drive/folders/0B4Q8pzCpDS_ecE81OG9nSTJvMUU?usp=sharing

**Comment by hannahkane** [comment]
Both are nice options. I have a slight preference for the photo on the right—it has good energy, and I feel like we use a lot of "people gathered around a laptop" photos, so it's actually kind of refreshing for me to see a different take on this. But I defer to @sabrinang and other designers!

**Comment by sabrinang** [comment]
I was just cautious if the right photo had a bottle, it may look like an alcoholic beverage? but i also like his hand up and smiling energy!

**Comment by hannahkane** [comment]
hmm....does kind of look like alcohol. maybe best to stick with the left!

**Comment by acabunoc** [comment]
While it probably doesn't affect the decision, the bottle is club mate! Hacker drink of Berlin

![image](https://cloud.githubusercontent.com/assets/617994/23776941/60a98ede-04ff-11e7-85c6-e039fd49b6d5.png)

**Comment by sabrinang** [comment]
What about if it is cropped?
![landing page - leadership training copy](https://cloud.githubusercontent.com/assets/2492510/23777012/a4db2cf2-04e6-11e7-91e3-61df3f3aa17e.jpg)
LOL Thanks @acabunoc for clarifying!

**Comment by hannahkane** [comment]
It's decided then. We'll use this as the photo:
![image](https://cloud.githubusercontent.com/assets/8389648/23777127/5c90c762-04e7-11e7-8b2e-4eb80ffcda9f.png)

Just kidding. Let's use the cropped version! Such good energy. :)

**Comment by sabrinang** [comment]
image included in mockup: https://invis.io/ZJ9YTAGEA#/222644515_Landing-Page-Leadership-Training
